### PR TITLE
Feature/#28 add option to use with existing project

### PR DESCRIPTION
**Pull request by janwidmer**

### CODE REVIEW

File: projects/spartacus-mock/src/lib/handlers/base-handler.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-import { currencyList } from '../mock-data/general/currencies';
-import { titleList } from '../mock-data/general/titles';
-import { countryList } from '../mock-data/general/countries';
-import { regionList } from '../mock-data/general/regions';
-import { consentTemplateList } from '../mock-data/consent-templates/consent-templates';
+import { consentTemplateList } from '../mock-data';
+import { countryList } from '../mock-data';
+import { currencyList } from '../mock-data';
+import { regionList } from '../mock-data';
+import { titleList } from '../mock-data';
```
Comment (claudio147) [inline projects/spartacus-mock/src/lib/handlers/base-handler.ts:7]: Could we combine them in a single import?

File: projects/spartacus-mock/src/lib/handlers/account-handler.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { ResponseComposition, rest, RestContext, RestHandler, RestRequest } from 'msw';
-import { savedCartResult } from '../mock-data/account/saved-cart';
-import { createAddress } from '../mock-data/account/addresses';
-import { createUser } from '../mock-data/auth/user';
-import { createConsentTemplate } from '../mock-data/consent-templates/consent-templates';
+import { ResponseComposition, RestContext, RestHandler, RestRequest, rest } from 'msw';
+import { createAddress } from '../mock-data';
+import { savedCartResult } from '../mock-data';
+import { createUser } from '../mock-data';
+import { createConsentTemplate } from '../mock-data';
```
Comment (claudio147) [inline projects/spartacus-mock/src/lib/handlers/account-handler.ts:5]: Could we combine them in a single import?

File: projects/spartacus-mock/src/lib/handlers/product-handler.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { ResponseComposition, rest, RestContext, RestHandler, RestRequest } from 'msw';
-import { productReferenceList } from '../mock-data/products/product-references';
-import { createReview, reviewList } from '../mock-data/products/product-reviews';
-import { createFullProduct } from '../mock-data/products/product';
-import { productSearchPage } from '../mock-data/search/search';
+import { ResponseComposition, RestContext, RestHandler, RestRequest, rest } from 'msw';
+import { createFullProduct } from '../mock-data';
+import { productReferenceList } from '../mock-data';
+import { createReview, reviewList } from '../mock-data';
+import { productSearchPage } from '../mock-data';
```
Comment (claudio147) [inline projects/spartacus-mock/src/lib/handlers/product-handler.ts:5]: Could we combine them in a single import?

File: projects/spartacus-mock/src/lib/handlers/user-handler.ts [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-import { authRevoke, createAuthToken } from '../mock-data/auth/auth';
-import { createUser } from '../mock-data/auth/user';
+import { ResponseComposition, RestContext, RestHandler, RestRequest, rest } from 'msw';
+import { addressList } from '../mock-data';
+import { customerCouponSearchResult } from '../mock-data';
+import { notificationPreferenceList } from '../mock-data';
+import { paymentDetailsList } from '../mock-data';
+import { productInterestSearchResult } from '../mock-data';
+import { authRevoke, createAuthToken } from '../mock-data';
+import { createUser } from '../mock-data';
```
Comment (claudio147) [inline projects/spartacus-mock/src/lib/handlers/user-handler.ts:8]: Could we combine them in a single import?

File: projects/spartacus-mock/src/lib/mock-data/account/product-interests.ts [hunk 5]
```diff
@@ -1,8 +1,8 @@
-import { Occ } from '@spartacus/core';
 import { faker } from '@faker-js/faker';
-import { createPagination } from '../general/pagination';
-import { createSort } from '../general/sort';
-import { createFullProduct } from '../products/product';
+import { Occ } from '@spartacus/core';
+import { createPagination } from '../general';
+import { createSort } from '../general';
```
Comment (claudio147) [inline projects/spartacus-mock/src/lib/mock-data/account/product-interests.ts:4]: Could we combine them in a single import?

**Review by claudio147 — CHANGES_REQUESTED** [review]

File: projects/spartacus-mock/src/lib/mock-data/account/product-interests.ts (hunk 5 above)
Comment (janwidmer) [reply projects/spartacus-mock/src/lib/mock-data/account/product-interests.ts:4]: done

File: projects/spartacus-mock/src/lib/handlers/account-handler.ts (hunk 2 above)
Comment (janwidmer) [reply projects/spartacus-mock/src/lib/handlers/account-handler.ts:5]: done

File: projects/spartacus-mock/src/lib/handlers/product-handler.ts (hunk 3 above)
Comment (janwidmer) [reply projects/spartacus-mock/src/lib/handlers/product-handler.ts:5]: done

**Review by claudio147 — APPROVED** [review]

**Pull request merged**